src/main/java/net/nikr/eve/jeveasset/gui/tabs/stockpile/StockpileTab.java

```diff
@@ -418,6 +411,11 @@ public void updateCache() {
 		filterControl.createCache();
 	}
 
+	@Override
+	public Collection<LocationType> getLocations() {
+		return new ArrayList<>(); //LocationsType
+	}
+
```

Comment:
I've noticed this in a couple places. Should we change getLocations() in the parent from abstract to this, and in the classes that actually have location data override?

Reply:
Personally I prefer to have the function like that, so when creating a new tool, you will be forced to decide what it's going to return.
But, if you think it's better to just have a default method, that is easy to change

Reply:
That's fine. Thought it might cut down on the boilerplate code a little but having to make a decision means you won't forget to implement it if it's relevant.